model/trigger/process.go

```diff
@@ -15,6 +15,38 @@ import (
 // same time. If the notifications array is not nil, they are valid and should
 // be processed as normal.
 func NotificationsFromEvent(e *event.EventLogEntry) ([]notification.Notification, error) {
+	h := registry.eventHandler(e)
+	if h != nil {
```

Comment:
nit: I would return an error if h is nil so that you don't need to indent this whole block

Reply:
See earlier comment about cruft cleaning, this is gone now